index.js

```diff
@@ -394,7 +373,7 @@ async function run () {
 
     try {
       core.info('Making prewarm application call...')
-      await makePrewarmRequest(url)
+      await makePrewarmRequest(`${url}/dashboard`)
```

Comment:
```suggestion
      await makePrewarmRequest(`${url}`)
```

It's correct to prewarm on the root.